src/ray/core_worker/transport/direct_actor_transport.cc

```diff
@@ -507,6 +507,9 @@ void CoreWorkerDirectTaskReceiver::HandleTask(
         }
       }
       if (task_spec.IsActorCreationTask()) {
+        auto concurrency_groups = task_spec.ConcurrencyGroups();
```

Comment:
This statement is not necessary. The variable could be passed into constructor directly.